source/_posts/progress-update-on-our-machine-learning-jump-start-course.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Progress update on our Machine Learning Jump Start course
+date: 2018-03-21 12:00:00
+tags: announcement, course
+---
+
+Last year we [announced to build a fresh, beginner-friendly Machine Learning course](/2017/12/15/machine-learning-jump-start-online-course/) with the promise to make this exciting field more accesible for starters. Many people want to get started with Machine Learning but have yet to find a course that isn't overloaded by heavy math and science. Our Machine Learning Jump Start breaks with that by explaing concepts in simple terms and focusing on easy to follow, practical online labs.
+
+<!-- more -->
+
+The course has been in [pre-sale](http://course.machinelabs.ai) since then and it's been very motivating for us to see all the excitment about the course from people all around the world.
+
+The format of the course is mainly video / screencast driven but also includes interactive labs that run entirely on the MachineLabs platform.
+
+Today we like to share an update about our progress and the current timeline for the course.
+
+Before we dive into this, enjoy this little teaser video that we recorded for you at our last company offsite in Oslo earlier this year.
+
+INTRO_VIDEO_HERE
+
+While we consider the course a long-term effort and plan to update the courseware continuously, our initial plan was to get the Early Access Preview (EAP) out this month.
```

Comment:
>  [...] and plan to update the <s>courseware</s> it continuously,

> [...] get the Early Access Preview (EAP) out this month (April 2018).

^ This last one is mainly for people who'll read the post much later and don't pay attention to the publish date of the post.